pennylane_forest/qpu.py

```diff
@@ -46,6 +46,11 @@ class QPUDevice(QVMDevice):
         device (str): the name of the device to initialise.
         shots (int): number of circuit evaluations/random samples used
             to estimate expectation values of observables.
+         wires (Iterable[Number, str]): Iterable that contains unique labels for the
+            qubits as numbers or strings (i.e., ``['q1', ..., 'qN']``).
+            The number of labels must match the number of qubits accessible on the backend.
+            If not provided, qubits are addressed as consecutive integers ``[0, 1, ...]``, and their number
+            is inferred from the backend.
```

Comment:
>If not provided, qubits are addressed as consecutive integers ``[0, 1, ...]``, and their number is inferred from the backend.

This will keep backcompatibility 👌